app/scrapers/scraper.rb

```diff
@@ -107,4 +108,14 @@ def scrape_links(result)
   def scrape_tags(result)
     []
   end
+
+  # Indicates whether this scraper result reliable and thus can be shown.
+  # 
+  # @param result [Hash] client response
+  # @info: Can be used to filter irrelevant / unpopular results, such as b-movies.
+  # @return [Boolean] indicates whether this scraper result is reliable.
+  #   By default every scraped result is reliable.
+  def reliable?(result)
+    true
```

Comment:
we wanted to white-list the results of scrapers that do not implement this method (i.e. accept every result).